src/pipe.js

```diff
@@ -203,6 +211,7 @@
         onAfterInit: assignHook('onAfterInit'),
         onDone: assignHook('onDone'),
         addPerfEntry: addPerfEntry,
+        addTTFMPEntry: addTTFMPEntry,
```

Comment:
I think `addPerfEntry.bind(null, 'ttfmp')` is shorter than the current implementation

Reply:
Bind is slow in older versions of v8 (older browsers might suffer from this). Thats why i went for explicit function. But, its just one time call so shouldn't matter much.